crates/matrix-sdk/src/client.rs

```diff
@@ -317,7 +317,26 @@ impl Client {
         *homeserver = homeserver_url;
     }
 
-    async fn get_supported_versions(&self) -> HttpResult<get_supported_versions::Response> {
+    /// Get the versions supported by the homeserver.
+    ///
+    /// This method should be used to check that a server is a valid Matrix
+    /// homeserver.
+    ///
+    /// # Example
+    /// ```no_run
+    /// # use futures::executor::block_on;
+    /// # block_on(async {
+    /// use matrix_sdk::{Client};
+    /// use url::Url;
+    ///
+    /// let homeserver = Url::parse("http://example.com").unwrap();
```

Comment:
Please no new unwraps in examples. You can use anyhow so the example can return any error.